docs/howto/manage-12-factor-app-charms.rst

```diff
@@ -0,0 +1,342 @@
+.. _howto-manage-12-factor-app-charms
+
+Manage 12-factor app charms
+***************************
+
+.. raw:: html
+
+   <!--TODO: There are a few more low-hanging fruit topics that we can and should cover:
+
+   When we create such sections we should include thinks at both the rock and the charm level (because the 12-Factor app app charm wants you to think about them in this coupled fashion). However, for the Rockcraft things, we should have those sections point to Rockcraft docs. For example:
+
+   ## Update the OCI image for a 12-Factor app charm
+
+   See {ref}`Rockcraft Documentation > How to update the OCI image... <15018md>`
+
+   -->
+
+.. card:: See also
+    :link: https://juju.is/docs/juju/charmed-operator
+
+    ``juju`` | 12-factor app charms
```

Comment:
I'm not jazzed by using cards, for several reasons. I think we can make do by making this a note or a plain paragraph, which is closer to the original structure (blockquote). That way we won't need to introduce new patterns and directives.

Longer-term I'd like to come up with smoother wording for introducing these links. Dressing them up as notes will be a good way to encourage me to correct them, since you know I don't like notes. :)

Reply:
I revisited this question with Dora. Let's instead use block quotes (two prepended spaces) for these.

That's also a structure we don't use often, so I'll easily be able to pick it up in later reviews/rewrites.